docs/tech/production-readiness.md

```diff
@@ -0,0 +1,49 @@
+# Production Readiness
+Production readiness documentation for BDK-based applications.
+
+## Logging
+The Symphony BDK setup your logger [MDC](http://logback.qos.ch/manual/mdc.html) (Mapped Diagnostic Context) with a value
```

Comment:
The symphony BDK **sets up** your logger

Reply:
Fixed https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/commit/6c7f74e7f581afd288f2b0cbfe89996a92fa02f9